src/vos/tests/vts_pm.c

```diff
@@ -2728,6 +2728,68 @@ uncommitted_parent(void **state)
 	start_epoch = epoch + 1;
 }
 
+static void
+test_uncommitted_key(void **state)
+{
+	struct io_test_args *arg = *state;
+	int                  rc  = 0;
+	daos_epoch_range_t   epr;
+	daos_key_t           dkey;
+	daos_key_t           akey;
+	daos_iod_t           iod;
+	d_sg_list_t          sgl;
+	char                 buf[32];
+	daos_epoch_t         epoch = start_epoch;
+	daos_handle_t        coh;
+	char                *first = "Hello";
+	char                 dkey_buf[UPDATE_DKEY_SIZE];
+	char                 akey_buf[UPDATE_AKEY_SIZE];
+	daos_unit_oid_t      oid;
+	struct dtx_id        xid;
+
+	test_args_reset(arg, VPOOL_SIZE);
+	coh = arg->ctx.tc_co_hdl;
+
+	memset(&iod, 0, sizeof(iod));
+
+	rc = d_sgl_init(&sgl, 1);
+	assert_rc_equal(rc, 0);
+
+	/* Set up dkey and akey */
+	oid = gen_oid(arg->ofeat);
+	vts_key_gen(&dkey_buf[0], arg->dkey_size, true, arg);
+	set_iov(&dkey, &dkey_buf[0], arg->ofeat & DAOS_OF_DKEY_UINT64);
+	vts_key_gen(&akey_buf[0], arg->akey_size, true, arg);
+	set_iov(&akey, &akey_buf[0], arg->ofeat & DAOS_OT_AKEY_UINT64);
```

Comment:
The unlcean-cherry-pick is due to the change in object class code which changed this API.